v3/packages.json

```
@@ -5240,6 +5268,78 @@
         { "filename": "exa/立方体(カメラ制御).exa", "archivePath": "exa/" }
       ]
     },
+    {
+      "id": "nilpo/ResizeFilter",
+      "name": "リサイズプラグイン",
+      "overview": "高品質なリサイズ（動画サイズ変更）フィルタ",
+      "description": "SIMD化やマルチコア対応による高速なリサイズが可能です。【動作にはVisual C++ 再頒布可能パッケージ 2008 x86(32bit)のインストールが必要です https://www.microsoft.com/ja-jp/download/details.aspx?id=26368 】また、AVX版やXOP版を使用する場合、【動作にはVisual C++ 再頒布可能パッケージ 2010 x86(32bit)のインストールが必要です https://www.microsoft.com/ja-JP/download/details.aspx?id=26999 】",
+      "developer": "nilpo",
+      "pageURL": "https://www.vector.co.jp/soft/winnt/art/se491671.html",
+      "downloadURLs": [
+        "https://www.vector.co.jp/soft/dl/winnt/art/se491671.html"
+      ],
+      "latestVersion": "111020",
+      "files": [
+        {
+          "filename": "plugins/ResizeFilter.auf",
+          "archivePath": "ResizeFilter"
+        },
+        {
+          "filename": "plugins/DLL/sse2filter.dll",
+          "archivePath": "ResizeFilter/DLL"
+        },
+        {
+          "filename": "plugins/DLL/sse2filter_yuy2.dll",
+          "archivePath": "ResizeFilter/DLL"
+        },
+        {
+          "filename": "plugins/DLL/wndfuncs.dll",
+          "archivePath": "ResizeFilter/DLL"
+        },
+        {
+          "filename": "plugins/DLL/avxfilter.dll",
+          "archivePath": "ResizeFilter/AVX"
+        },
+        {
+          "filename": "plugins/DLL/xopfilter.dll",
+          "archivePath": "ResizeFilter/XOP",
```

```suggestion
          "archivePath": "ResizeFilter/XOP"
```
Sorry for the trailing comma.